Stratis.Bitcoin/Utilities/CustomThreadPoolScheduler.cs

```diff
@@ -1,164 +0,0 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Stratis.Bitcoin.Utilities
-{
-    /// <summary>
-    /// Task scheduler that spawns an exact number of threads for execution of its tasks.
-    /// </summary>
-    public class CustomThreadPoolTaskScheduler : TaskScheduler, IDisposable
```

Comment:
I guess we don't need this class anymore!

Reply:
hope so :-)